api/filter_dimension_options.go

```diff
@@ -27,14 +30,30 @@ func (api *FilterAPI) getFilterBlueprintDimensionOptionsHandler(w http.ResponseW
 	ctx := r.Context()
 	log.Event(ctx, "get filter blueprint dimension options", log.INFO, logData)
 
+	// get limit from query parameters, or default value
+	limit, err := getPositiveIntQueryParameter(r.URL.Query(), "limit", api.defaultLimit)
+	if err != nil {
+		log.Event(ctx, "failed to obtain limit from request query paramters", log.ERROR, logData)
+		setErrorCode(w, err)
+		return
+	}
+
+	// get offset from query parameters, or default value
+	offset, err := getPositiveIntQueryParameter(r.URL.Query(), "offset", api.defaultOffset)
+	if err != nil {
+		log.Event(ctx, "failed to obtain offset from request query paramters", log.ERROR, logData)
```

Comment:
typo in `parameters`
```suggestion
		log.Event(ctx, "failed to obtain offset from request query parameters", log.ERROR, logData)
```